package.json

```diff
@@ -19,15 +19,13 @@
     "link:emails:js": "rm -rf emails; ln -s packages/cli/src/generator_templates/js/emails emails;",
     "build": "preconstruct build",
     "test": "yarn link:emails; yarn jest",
+    "test:integration": "./scripts/assert-free-ports.sh; MAILING_CI=true MAILING_DATABASE_URL=$MAILING_DATABASE_URL_TEST WEB_DATABASE_URL=$WEB_DATABASE_URL_TEST start-test ci:cli :3883 ci:web :3000 \"yarn link:emails; yarn jest -c jest.integration.config.ts $TEST_FLAGS\"",
```

Comment:
nice, I didn't know how to do this with the quotes, so that's why i moved it to the "unsafe" methods.  this is good that you can't run those commands without running the wrapping function that sets the env variables.  the point is to prevent you from running the truncate commands in the integration test setup on your dev database by accident.